bin/pint

```diff
@@ -150,22 +151,94 @@ if command_args['--region']:
 
 try:
     if command_args['images']:
-        data = ifsrequest.get_image_data(
-            framework,
-            image_state,
-            output_format,
-            region,
-            command_args['--filter'])
-        if '<images/>' in data or '"images": []' in data:
-            data = 'No information available. Please check your filter'
-        print(data)
+        info = False
+        no_info = ''
+        data_json = {}
+        data_xml = None
+        for reg in region.split(','):
+            data = ifsrequest.get_image_data(
+                framework,
+                image_state,
+                output_format,
+                reg.strip(),
+                command_args['--filter']
+            )
+            if '<images/>' in data or '"images": []' in data:
+                info = False
+                continue
+
+            info = True
+            # there is info in some region/s
+            # so we show the empty response/s
+            # and not the message
+            if output_format in output_options:
+                if 'json' in output_format:
+                    data = json.loads(data)
+                    data_json['images'] = data_json.get('images', []) + data.get('images')
+                    data = data_json
+                else:
+                    if data_xml is None:
+                        data_xml = etree.Element('images')
+
+                    data = etree.fromstring(data.encode('utf-8'))
+                    data_xml.extend(data)
+                    data = etree.tostring(
+                        data_xml,
+                        xml_declaration=True,
+                        encoding='UTF-8',
+                        pretty_print=True
+                    ).decode()
+
+        if not info:
+            print("No information available. Please check your filter")
```

Comment:
No message, we'd even print that message if the user provides no filter, i.e. ```pint amazon images --active --region a-new-region-with-no-images-yet``` would still produce a message telling the user to check the filter when they applied no filter.